test/isochrone.cc

```diff
@@ -94,16 +103,27 @@ void test_isochrones() {
 // Test auto isochrone with one contour
 // 32bit builds fail in release mode we'll look at this separately
 #if _WIN64 || __amd64__
```

Comment:
> // 32bit builds fail in release mode we'll look at this separately

Is this still the case?

Reply:
Yup, segmentation fault on 32bit x86... https://circleci.com/gh/valhalla/valhalla/17069

Reply:
Opened issue https://github.com/valhalla/valhalla/issues/2187